docs/reference/adventureworks/armTemplates/auxiliary/policies.json

```diff
@@ -10677,6 +10677,12 @@
                     "field": "type",
                     "equals": "Microsoft.Network/virtualNetworks/subnets"
                   },
+                  {
+                    "not": {
+                      "field": "name",
+                      "equals": "AzureBastionSubnet"
```

Comment:
should we parameterize the value (AzureBastionSubnet) and change this to be "in"?

Reply:
Lets discuss on our call later in the week. But I agree and think we should likely make this an Array so its more flexible for other subnets

Reply:
This has now been done in the latest commits